app/main.py

```diff
@@ -1,13 +1,22 @@
 def format_linter_error(error: dict) -> dict:
-    # write your code here
-    pass
+    return {
+        "line": error["line_number"],
+        "column": error["column_number"],
+        "message": error["text"],
+        "name": error["code"],
+        "source": "flake8"
+    }
 
 
 def format_single_linter_file(file_path: str, errors: list) -> dict:
-    # write your code here
-    pass
+    return {
+        "errors": [format_linter_error(error)for error in errors],
```

Comment:
Add a space between `format_linter_error(error)` and `for`. Besides that, an alternative approach to write this expression using `map` would be the following: 
```suggestion
        "errors": list(map(format_linter_error, errors)),
```